src/main.cc

```diff
@@ -135,7 +133,11 @@ static int
 on_shutdown_f(va_list ap)
 {
 	(void) ap;
-	trigger_run(&box_on_shutdown_trigger_list, NULL);
+	/** Time to wait for shutdown trigger finished */
+	const double shutdown_trigger_wait_time = 3.0;
```

Comment:
I'd better make this constant global or even anonymous enum (see MEMTX_DDL_YIELD_LOOPS, TUPLE_INDEX_BASE etc).

Reply:
Also what happens if shutdown trigger is timed out? Please add description to the docbot request